frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -21,7 +21,7 @@ type UICastRow = {
 
 type UICastPosition = {
   pos: Position;
-  castSize: number;
+  dancerCount: number;
```

Comment:
Is the intention to rename `UICastPosition` to `UIDancerPosition` and `castRows: UICastRow[]` to `dancerRows: UIDancerRow[]`?

Or is there an intentional difference here between a `cast` and a `dancer`?

Reply:
No, the purpose is to clarify an ambiguity. In the past, the number of casts was was a fluid number that was stored indirectly (dancers were stored in casts and the # of casts was deduced by in what casts the dancers had been stored.

When I changed the cast logic, I used castSize to specify how many dancers the cast had for a certain position. However, when the customer wanted to be able to change # of casts dynamically, we needed to keep track of #casts, so I added the field castCount. castCount and castSize was confusing (I confused them myself more than once) so I felt that changing castSize to dancerCount, which it really was, was easier to pair with castCount than the old castSize.

In the process of changing the cast logic, I changed how the cast is stored in memory. In the past there was a closer fit between how casts were stored in the database and how they were stored in memory than we have now. It is something that we may want to revisit when we clean up the code.

Reply:
As a non-subject-matter-expert, to me "dancerCount" sounds exactly the same as "castSize" and "castCount", so it just looks like a typo that `UICastPosition` has `castRows` but `dancerCount`. 

But if you think it's more clear, we don't need to do anything else here right now.